src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,56 @@
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get('sex') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function setSearchWidth(params: any) {
```

Comment:
Using 'any' type for the 'params' argument is not recommended as it defeats the purpose of TypeScript's type safety. Consider defining a specific type for 'params' to ensure better type checking.